### PR TITLE
Add error handling and some refactor.

### PR DESCRIPTION
**Pull request by tnv1**
Add missing check error handling and some refactors code.

### CODE REVIEW

File: x/feeabs/keeper/config.go [hunk 1]
```diff
@@ -59,15 +59,16 @@ func (k Keeper) SetHostZoneConfig(ctx sdk.Context, chainConfig types.HostChainFe
 }
 
 func (k Keeper) DeleteHostZoneConfig(ctx sdk.Context, ibcDenom string) error {
```
Comment (nghuyenthevinh2000) [inline x/feeabs/keeper/config.go:61]: need to add a test case for reviewing

File: x/feeabs/types/build_memo_test.go [hunk 2]
```diff
@@ -16,7 +16,7 @@ func TestParseMsgToMemo(t *testing.T) {
 	}
 
 	swap := types.Swap{
-		OutPutDenom: "khanhyeuchau",
+		OutputDenom: "khanhyeuchau",
```
Comment (nghuyenthevinh2000) [inline x/feeabs/types/build_memo_test.go:19]: need to change denom

Comment (tnv1) [reply x/feeabs/types/build_memo_test.go]: Done

File: x/feeabs/keeper/config.go (hunk 1 above)
Comment (tnv1) [reply x/feeabs/keeper/config.go:61]: Done in #121

**Review by minhngoc274 — APPROVED** [review]

**Pull request merged**